app/Database/migrations/2022_12_27_192218_create_events.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+// Add events table and update flights & pirep tables for references
+return new class() extends Migration {
+    public function up()
+    {
+        // Create events table
+        Schema::create('events', function (Blueprint $table) {
+            $table->increments('id');
```

Comment:
Use `primary` for this

Reply:
Primary or Unique ?

Reply:
You want to replace `increments` with `primary` ?

https://laravel.com/docs/9.x/migrations#column-method-increments

Reply:
Looks like they changed that, well, keep it as `primary` so it matches the other migrations

Reply:
Ok no problem, I used `increments` to have the proper field and primary together :) Switching it now

Reply:
Had to use `integer` for the id field and added `primary` as the index